packages/create-sitecore-jss/src/common/prompts/proxy.ts

```diff
@@ -0,0 +1,52 @@
+import { Answers, DistinctQuestion } from 'inquirer';
+import { getDefaultProxyDestination } from '../utils/helpers';
+
+export const FALLBACK_PROXYNAME = 'sitecore-jss-proxy-app';
+
+/**
+ * Set of args needed for proxy prompt
+ */
+export type ProxyArgs = Pick<Answers, 'yes'> & {
+  // input args
```

Comment:
are these extra comments needed?

Reply:
I think so. The args/answers duality can get confusing when working with prompts logic. I try to differentiate the inputs and outputs used by this prompt.